### ISSUE TITLE
Xcode 9 Problem

### ISSUE THREAD

**Issue by SalvatoreAD** [XWebView/XWebView#90]
this is the error

`Instance member 'exclusion' cannot be used on type 'XWVMetaObject'`

in XWVMetaObject.swift

Can you help me?

**Comment by aahad** [comment]
I am facing the same issue, so far any quick fix if no update is available yet ? Thanks

**Comment by aahad** [comment]
I fixed it by adding below piece of code:

```
typealias Element = (key: String, value: XWVMetaObject.Member)    
subscript(position: Dictionary<String, XWVMetaObject.Member>.Index) -> (key: String, value: XWVMetaObject.Member) {
               return members[position]
}
```
Reference:  [XWebView/XWVMetaObject.swift](https://github.com/XWebView/XWebView/pull/86/files#diff-4c8d5e8d5be1cfa08d83b3178200e35e)

**Comment by xwv** [comment]
The fix is included in #91 
Thanks.
